app/backend/src/clj/teet/admin/index_queries.clj

```diff
@@ -0,0 +1,43 @@
+(ns teet.admin.index-queries
+  (:require [teet.db-api.core :as db-api :refer [defquery]]
+            [datomic.client.api :as d]
+            [teet.transit :as transit]
+            [teet.util.euro :as euro])
+  (:import (java.util UUID)))
+
+(defn- get-index-data
+  "Returns the indexes data"
+  [db]
+  )
+
+(defn- with-bigdec-format [x]
+  (transit/with-write-options
+    {java.math.BigDecimal euro/format-no-sign}
+    x))
+
+(def index-listing-attributes
```

Comment:
should probably be in a index-model namespace

Reply:
I'll leave it for the moment to the queries as it is only part of this query.